packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```diff
@@ -912,16 +926,11 @@ Please provide a valid TCP port (an integer between 0 and 65535, inclusive).
         ),
       );
     }
-    if (websocketUri != null) {
-      if (debuggingOptions.vmserviceOutFile != null) {
-        _fileSystem.file(debuggingOptions.vmserviceOutFile)
-          ..createSync(recursive: true)
-          ..writeAsStringSync(websocketUri.toString());
-      }
-      _logger.printStatus('Debug service listening on $websocketUri');
-    }
+
     appStartedCompleter?.complete();
-    connectionInfoCompleter?.complete(DebugConnectionInfo(wsUri: websocketUri));
+    if (!supportsServiceProtocol) {
+      connectionInfoCompleter?.complete(DebugConnectionInfo());
+    }
```

Comment:
Comment explaining why this is needed here, or maybe convert it to an `else` branch of the `if(supportsServiceProtocol)` above.

Reply:
Not sure why I did that... thanks for catching it!